openeo_udp/sentinel1_stats/README.md

```diff
@@ -1,4 +1,15 @@
-Calculate S1 SAR stats (min, max, mean, sd, q10, q50, q90). 
+# Description
 
-This service can cost an approximate of 3-5 credits per sq km. 
-This cost is based on resource consumption only and added-value cost has not been included.
+This algorithm derives temporal statistics from the Sentinel-1 GRD collection for the selected spatial- and temporal extent of interest. In order to callibrate the SAR backscatter signal, `sigma0-ellipsoid` has been applied within this workflow. 
+
+
+# Performance characteristics
```

Comment:
4 credits is the minimal cost for a UDP, I do not believe it should be part of this PR, but I should increase the size of the benchmarking to get a clearer sight on the 'scaled up' cost